UnitySDK/Assets/ML-Agents/Examples/GridWorld/Scenes/GridWorldRenderTexture.unity

```diff
@@ -0,0 +1,1301 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
```

Comment:
Not a fan of having 2 gridworld scenes now... This adds confusion and testing requirements.

Reply:
Would it be preferable to have a new example, or no example at all?